### ISSUE TITLE
Parse error when Installation

### ISSUE THREAD

**Issue by TsichiChang** [sorin-ionescu/prezto#1268]
Step 3, I have this error:

![0c79a45e-ec1a-4532-83d8-3c68fbee066c](https://cloud.githubusercontent.com/assets/5811211/24138237/667334a8-0e52-11e7-8355-d61414e5001f.png)

My zsh's version number is 5.2.

Any suggestions?

**Comment by jostyee** [comment]
GitHub renders README wrongly, try:
```shell
setopt EXTENDED_GLOB
for rcfile in "${ZDOTDIR:-$HOME}"/.zprezto/runcoms/^README.md(.N); do
  ln -s "$rcfile" "${ZDOTDIR:-$HOME}/.${rcfile:t}"
done
```

**Comment by TsichiChang** [comment]
Thanks, you save my life.

**Comment by paulmelnikow** [comment]
Note: This was fixed in #1271.
